data/huion-kamvas-pro-24.tablet

```diff
@@ -0,0 +1,60 @@
+# Huion
+# Kamvas Pro 24
+# GT-240
+#
+# sysinfo.0LZKOwtz0e.tar.gz
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/351
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#          *-----------------------*
+#    A     |                       |     K
+#    B     |                       |     L
+#    C     |                       |     M
+#    D     |                       |     N
+#    E     |                       |     O
+#          |        DISPLAY        |
+#    F     |                       |     P
+#    G     |                       |     Q
+#    H     |                       |     R
+#    I     |                       |     S
+#    J     |                       |     T
+#          *-----------------------*
+#
+# Touch Strip Map:
+# (A=1st strip, B=2nd strip, ...)
+#
+#
+#    *-----------------------*
+#    |                       |
+#  A |                       | B
+#  a |        DISPLAY        | b
+#    |                       |
+#    *-----------------------*
+#
+# Also matches:
+# GAOMON S56K which doesn't have buttons but since HUION re-uses usb IDs
+# there is nothing we can do about this.
+
+[Device]
+Name=Huion Kamvas Pro 24
+ModelName=GT-240
+Class=Cintiq
+DeviceMatch=usb:256c:006e:Tablet Monitor Pen;usb:256c:006e:Tablet Monitor Pad;usb:256c:006e:Tablet Monitor Touch Strip;
+Width=26
+Height=15
+Layout=kamvas-pro-24.svg
+Styli=@generic-no-eraser;
+IntegratedIn=Display
+
+[Features]
+Buttons=20
+Reversible=false
+Ring=false
+Stylus=true
+Touch=false
+
+[Buttons]
+Left=A;B;C;D;E;F;G;H;I;J
+Right=K;L;M
```

Comment:
libwacom is a static database and doesn't know how/if the device works. It provides information we can't get from the kernel which in this case means: please add all buttons so the information is correct OOTB. If the device starts working in the future we don't need to update this database again.

This goes for the SVG too, it'll need to feature all buttons even if right now they won't work.

Reply:
Done on the tablet side. I'll create the SVG file later today.